docs/.vuepress/config.js

```diff
@@ -162,6 +162,7 @@ module.exports = {
   },
   chainWebpack(config, isServer) {
     config.resolve.alias.set("$vue", "vue/dist/vue.esm.js");
+    config.resolve.alias.set("@rei/cdr-tokens$", "@rei/cdr-tokens/dist/js/cdr-tokens.esm.js");
```

Comment:
this is needed because vuepress does some weird webpack business causing the token imports in modal to break.